src/main/java/io/github/lottetreg/echo/Connection.java

```diff
@@ -20,6 +21,15 @@ public InputStream getInputStream() {
     }
   }
 
+  public OutputStream getOutputStream() {
+    try {
+      return socket.getOutputStream();
+    } catch(IOException e) {
+      System.out.println(e);
+      return null;
+    }
```

Comment:
I commented on the wrong PR earlier - is the exception handling going to come in a future PR?

Reply:
@klkelley yes it is! :)